### PR TITLE
Use translations for new about page

### PR DESCRIPTION
**Pull request by renderf0x**
Adds translations to the new about page added in #251 

### How to test

Use netlify preview or pull down and `yarn install`. Page should look EXACTLY like prod.

### CODE REVIEW

**Comment by renderf0x** [comment]
One note on a choice I've made here...

You'll notice that there are some strings in here that looks something like this `"about.timeline.items.1.text.0",`, and we're iterating over items containing those (ie, the indexes are explicit in the string ids).

Why couldn't we just substitute the indexes in there, you ask. We can. However, I chose to be 100% explicit here in what's going on, as there's too much indirection for my taste if we do that. At some point, you start to wonder what you're iterating over, as in one context it's the items in the data list from the component, and in another it's the string ids.

I opted to write this for readability and debuggability instead of saving a few characters and being 100% DRY.

**Comment by renderf0x** [comment]
Note that this is also our first use of the slotted, interpolated `i18n` component.
[You can find details on it here](https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage)

File: components/about/SiteTimeline.vue [hunk 1]
```diff
@@ -38,18 +40,24 @@ export default {
...
       },
       {
-        date: "Presently",
-        dateHeadline: "A team of volunteers",
-        text: "Find a Doc is community-driven with the support of a team of volunteers. Currently, the team is focused on keeping the immigrant community informed about clinics offering waiting lists for vaccinations to reduce waste. <br> In the future, we want Find a Doc to help non-Japanese speaking residents of Japan find clinics that offer services in their native language.",
-        textHeadline: "You can fill the void by citizen action.",
+        date: "about.timeline.items.1.date",
+        dateHeadline: "about.timeline.items.1.dateHeadline",
+        text: [
+          "about.timeline.items.1.text.0",
```
Comment (ann-kilzer) [inline components/about/SiteTimeline.vue:55]: It's a new pattern to me, but happy to try it out!

**Review by ann-kilzer — APPROVED** [review]
Looking really good. Could you please address the linting failures before we merge?

**Comment by renderf0x** [comment]
> Looking really good. Could you please address the linting failures before we merge?

Yep, needed some additions to both the lint config, and a few small fixes.

**Pull request merged**